packages/colibri/src/project_manager/project_manager.ts

```diff
@@ -501,8 +505,21 @@ export class Project_manager {
 
         return await this.tools_manager.get_test_list(this.get_project_definition(n_config_manager));
     }
-}
-
 
+    public getBuildSteps(): t_taskRep[] {
+        return [];
+    }
 
+    // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
+    public runTask(_taskType: e_quartusTaskType, _callback: (result: p_result) => void): ChildProcess {
+        return {} as ChildProcess;
+    }
 
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    public async cleallAllProject(): Promise<t_loader_action_result> {
```

Comment:
why not raise an exception? and this should be part of an interface or an abstract method

Reply:
Added also to ProjectManager generic.

I have changed the implementation. Now it returns a chilc_exec object. Changed async by sync